Dockerfile

```diff
@@ -23,5 +23,4 @@ RUN echo "deb [signed-by=/usr/share/keyrings/onlyoffice.gpg] http://download.onl
     apt-get -y update && \
     apt-get -y install onlyoffice-documentbuilder
 CMD /bin/bash -c "onlyoffice-documentbuilder -v; \
-                  cd /doc-builder-testing; \
-                  rake"
+                  cd /doc-builder-testing"
```

Comment:
Do we really need to change it?
Correct me if I wrong, but I think the docker run command from `.github/workflows/units.yml` https://github.com/ONLYOFFICE/doc-builder-testing/pull/1169/files#diff-2696f6c07f1d42e08655adbf72219740095fee482ee8da226f8f7a4e2659fce7R11
will work without modification of existing file

Reply:
If I understood the question correctly, then yes, I want to explicitly pass the parameter to the container to run the functionality via rake

I also suggest that we separate the receipt of the builder version into a separate step

```dockerfile
RUN onlyoffice-documentbuilder -v
```

And run dockerfile with implemented good practice to run parameters in the container

ENTRYPOINT startup context
CMD startup parameter

```dockerfile
ENTRYPOINT  /bin/bash -c 
CMD ["rake"]
```

like
https://github.com/palladium-tms/palladium-view/blob/48b9eba7db58d622f21e8ee5f57209584bf4ed60/docker-build/Dockerfile#L9

This will allow you to send rake commands to the assembled container in the future

Reply:
https://docs.docker.com/reference/build-checks/json-args-recommended/

Reply:
https://docs.docker.com/build/building/best-practices/#entrypoint

Reply:
Actually as far as I remember `onlyoffice-documentbuilder -v` is leftover from very old times when the documentbuilder do not run correctly on first time after installation

As far as it fixed - we can remove the line with showing version all together

But yeah, having cmd like json is a good practice for sure